subsys/pcd/src/pcd.c

```diff
@@ -183,10 +208,11 @@ static int network_core_pcd_cmdset(const void *src_addr, size_t len, bool wait)
 		k_busy_wait(1 * USEC_PER_SEC);
 
 		err = pcd_fw_copy_status_get();
-	} while (err == PCD_STATUS_COPY);
+	} while (err == command_status);
```

Comment:
while we are here, could we rename `err` to something more precise? `err` makes the code harder to read.